cocos/2d/CCScene.cpp

```diff
@@ -209,6 +209,8 @@ void Scene::render(Renderer* renderer, const Mat4& eyeTransform)
         // And it is important that the change is "permament", because the matrix might be used for calculate
         // culling and other stuff.
         camera->setAdditionalTransform(eyeTransform.getInversed());
+        if (eyeProjection)
+            camera->setAdditionalProjection(*eyeProjection * camera->getProjectionMatrix().getInversed());
```

Comment:
I don't think so... because we need replace the projection matrix of every camera by `eyeProjection`, not a additional transform(this is why I used `*eyeProjection * camera->getProjectionMatrix().getInversed()`, not `*eyeProjection`), and we don't know the projection matrix of every camera before calling `Scene::render()`.